extensions/typescript/src/typescriptServiceClient.ts

```diff
@@ -197,17 +198,25 @@ export default class TypeScriptServiceClient implements ITypescriptServiceClient
 		this._checkGlobalTSCVersion = true;
 		this.trace = this.readTrace();
 		this.tsServerLogLevel = this.readTsServerLogLevel();
+		this.checkJs = this.readCheckJs();
+
 		disposables.push(workspace.onDidChangeConfiguration(() => {
 			let oldLoggingLevel = this.tsServerLogLevel;
```

Comment:
Managing config like this is becoming pretty old. I'll look into refactoring this logic to be clearer in another change